browsers/webpack.config.js

```diff
@@ -285,6 +286,7 @@ module.exports = (env, options) => ({
     new webpack.EnvironmentPlugin({
       // If not found, these values will be used as defaults
       DEBUG: !isProd(options),
+      REDUX_DEV_TOOLS: process.env.REDUX_DEV_TOOLS,
```

Comment:
The `defaults` object is the right one unless the flag depends on webpack’s `--mode` flag, which this does

```suggestion
      REDUX_DEV_TOOLS: !isProd(options),
```